kedro/templates/project/{{ cookiecutter.repo_name }}/src/tests/test_run.py

```diff
@@ -36,6 +36,7 @@
 To run the tests, run ``kedro test``.
 """
 from pathlib import Path
+from unittest.mock import patch
```

Comment:
Can we use `mocker.patch` instead? 🙂

Reply:
Done! Is the main benefit to not require the additional import/context manager?

Reply:
Fewer imports but also consistency with only using one library for testing. `pytest` has pretty much all the functionality that you can find in `unittest` so no point in mixing.